regulations/static/regulations/css/less/layout.less

```diff
@@ -4,6 +4,11 @@
  layout.less defines the structural layout of the site
  */
 
+// focus for accessibliity tabbing
+a:focus {
+    border: 1px dotted @black;
+}
```

Comment:
What do you think about modifying the rule in `normalize.less`? I'm not sure why the outline there isn't being applied